Tools/DumpRenderTree/mac/DumpRenderTree.mm

```diff
@@ -1827,6 +1827,13 @@ static void resetWebViewToConsistentState(const WTR::TestOptions& options, Reset
 
     WebCoreTestSupport::setAdditionalSupportedImageTypesForTesting(String::fromLatin1(options.additionalSupportedImageTypes().c_str()));
 
+#if ENABLE(VIDEO)
+    if (!options.captionDisplayMode().empty())
+        [mainFrame _setCaptionDisplayMode:[NSString stringWithUTF8String:options.captionDisplayMode().c_str()]];
+    else
+        [mainFrame _setCaptionDisplayMode:@"forcedOnly"];
```

Comment:
Don't you want `@"forcedonly"`?

Reply:
Done https://github.com/WebKit/WebKit/compare/322a92c6033af72d71b892d99ac0fe96aa933cbc..57043c4e93599302fc75190a5f42d2cd43822453